File: jawaban.py
print(20* "=", "TIPE DATA STRING", 20* "=")
print("String")
Data_string = "Halo, ini adalah string hengkel..."
print("Nilai dari Data_string =", Data_string)
print("-------------------")

print(20* "=", "TIPE DATA NUMERIK", 20* "=")

print("Integer (int)")
Data_int = 10
print("Nilai dari Data_Int =", Data_int)
print("-------------------")

print("float")
Data_float = 10.34
print("Nilai dari Data_float =", Data_float)
print("-------------------")

print("complex")
Data_complex = 2j
print("Nilai dari Data_complex =", Data_complex)
print("-------------------")

print(20* "=", "TIPE DATA BOOLEAN", 20* "=")
print("Boolean")
Data_boolean = True
print("Nilai dari Data_boolean =", Data_boolean)
print("-------------------")

print(20* "=", "TIPE DATA SEQUENCE", 20* "=")
print("list")
Data_list = ["Item1", 2, "Item lanjutan", 2.5]
print("List merupakan tipe data yang menyatakan kumpulan item-item")
print("Nilai dari Data_list =", Data_list)
print("-------------------")

print("Tuple")
Data_tuple = "Item1", 2, "Item lanjutan", 2.5
print("Tuple merupakan tipe data yang menyatakan kumpulan item-item. Hanya saja item didalam nya bersifat tetap, tdak bisa diubah")
print("Nilai dari Data_tuple =", Data_tuple )
print("-------------------")

print("Range")
Data_range = range(5)
print("""Range e merupakan tipe data yang digunakan untuk menghasilkan number (dan biasanya berhubungan juga dengan nomor Indeks) dari jangkauan tertentu
Range ini biasanya diaplikasikan pada loop""")
print("Nilai dari Data_range =", Data_range )
print("-------------------")


print(20* "=", "TIPE DATA SET", 20* "=")

print("Set")
Data_set = set("ABCDEFG")
print("""Set merupakan koleksi atau kumpulan data yang tak tersusun, dalamnya juga tidak boleh ada duplikasi
Isi dari set sendiri bisa berupa string, Integer, list, dan lainnya""")
print("Nilai dari Data_set =", Data_set)
print("-------------------")

print(20* "=", "TIPE DATA DICTIONARY", 20* "=")
print("Dictionary")
Data_dictionary = {
    "Nama"  : "Rizki",
    "NIM"   : "i.2210881"
} #Kata Kunci  : #Nilai nya 
print("""Dictionary merupakan tipe data yang digunakan untuk menyimpan suatu nilai data. Pada Dict terdapat kata kunci dan nilai dari kata kunci tsb. """)
print("Nilai dari Data_dictionary =", Data_dictionary)
print("-------------------")

print(50*"=")
print("Untuk mengecek tipe data bisa digunakan type(variabel ataupun nilai data nya secara langsung)")
print(type(Data_int))
print(type(Data_float))
print(type(Data_complex))
print(type(Data_boolean))
print(type(Data_list))
print(type(Data_tuple))
print(type(Data_range))
print(type(Data_set))
print(type(Data_dictionary))

File: jawaban.java
public class Jawaban {
  public static void main (String [] args){
    //Tipe Data Integer

    System.out.println("==========TIPE DATA INTEGER==========");
    byte nilai_byte = 127;
    short nilai_short = 32767;
    int nilai_int = 2147483647;
    long nilai_long = 9000000000L;
    
    System.out.println("Integer merupakan tipe data yang bernilai angka bilangan bulat, berikut tipe data bilangan bulat dan contoh nya :");
    System.out.println("Nilai dari tipe data byte = " + nilai_byte);
    System.out.println("Nilai dari tipe data short = " + nilai_short);
    System.out.println("Nilai dari tipe data int = " + nilai_int);
    System.out.println("Nilai dari tipe data long = " + nilai_long);
    System.out.println();
    
    //Floating Point (Desimal)
    System.out.println("==========TIPE DATA FLOATING POINT==========");
    float nilai_float = 2.44f;
    double nilai_double = 2.12345678910d;
    System.out.println("Floating point merupakan bilangan desimal. Pada tipe data ini ada 2, yaitu Float dan Double");
    System.out.println("Float untuk 6-7 desimal dan berukuran 32 bit, sedangkan Double untuk hingga 15 digit desimal dan ukurannya 64bit");
    System.out.println("-----------------------------");
    System.out.println("Contoh nilai float = " + nilai_float);
    System.out.println("Contoh nilai double = " + nilai_double);
    System.out.println();

    //CHAR
    System.out.println("==========TIPE DATA CHAR==========");
    char char_A = 'A';
    char char_B = 'B';  
    System.out.println("Nilai char_A = " + char_A);
    System.out.println("Nilai char_B = " + char_B);
    System.out.println();

    //BOOLEAN
    System.out.println("==========TIPE DATA BOOLEAN==========");
    boolean Life_Exist = true;
    boolean Life_Doesnt_Exist = false;
    System.out.println("Boolean merupakan tipe data True - False, biasanya digunakan untuk menyatakan suatu hal/kondisi bernilai Benar atau Salah");
    System.out.println("-------- Contoh --------");
    System.out.println("Nilai Boolean dari Life_Exist = " + Life_Exist);
    System.out.println("Nilai Boolean dari Life_Doesnt_Exist = " + Life_Doesnt_Exist);
    System.out.println();

    //STRING
    System.out.println("==========TIPE DATA STRING==========");
    String myString = "Hi There ! Saya hengkel --Contoh String di variabel myString";
    System.out.println(myString);
    System.out.println();
  }
}
